big_tests/tests/sm_SUITE.erl

```diff
@@ -1160,6 +1281,33 @@ no_crash_if_stream_mgmt_disabled_but_client_requests_stream_mgmt_with_resumption
 %%--------------------------------------------------------------------
 %% Helpers
 %%--------------------------------------------------------------------
+start_hook_listener(Resource) ->
+    TestCasePid = self(),
+    rpc(mim(), ?MODULE, rpc_start_hook_handler, [TestCasePid, Resource]).
+
+rpc_start_hook_handler(TestCasePid, User) ->
+    Handler = fun(Acc, Res) ->
+                  JID = mongoose_acc:to_jid(Acc),
+                  case {jid:nodeprep(User), jid:to_lus(JID)} of
```

Comment:
`jid:nodeprep(User)` may be moved outside fun definition, because we are already in MIM context.